### DISCUSSION TITLE
Why is for you unlike every other feature preview?

### DISCUSSION THREAD

**Discussion by xcfrg** [community/community#13271] [Other Feature Feedback, Questions, & Ideas]
The For you feed, just like every other option with the (Beta) tag, should be able to be disabled in feature preview as is for every other feature preview.

I do not want this feature, and like many others, would like this to be like every other feature preview. Why is this one special? Or more important?

I personally prefer a chronological order of events, as was the previous.

It is annoying that the default is now "For you" and there should be an option to permanently it, regardless of it's beta status.

**Comment by bdresser** [comment]
Hi @xcfrg! Your old feed remains exactly as-is under the "Following" tab. 

Since navigating to the "For you" tab is entirely optional, and since getting feedback from a wide range of users is super important here, we didn't launch this as part of Feature Preview.

**Reply by xcfrg** [reply]
> Hi @xcfrg! Your old feed remains exactly as-is under the "Following" tab.
> 
> Since navigating to the "For you" tab is entirely optional, and since getting feedback from a wide range of users is super important here, we didn't launch this as part of Feature Preview.

Thanks!, it was defaulting to "For you" before and I wasn't sure why, likely related to my browser.

**Reply by bdresser** [reply]
The homepage should load with whichever tab you had selected last. So if you select Following, it should remain on Following next time you load the page. Let us know if you see otherwise!

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
